openapi2/openapi2.go

```diff
@@ -43,7 +54,22 @@ func (swagger *Swagger) AddOperation(path string, method string, operation *Oper
 	pathItem.SetOperation(method, operation)
 }
 
+func (swagger *Swagger) AddPathExtensions(path string, extensionProps openapi3.ExtensionProps) {
```

Comment:
It looks like this is only called from the conversion package and could very well be implemented there.
How do you see this function being useful to other callers? Can we make this a private func within `openapi2_conv`?